nexmark-flink/src/main/java/com/github/nexmark/flink/workload/Workload.java

```diff
@@ -103,21 +103,12 @@ public long getWarmupEvents() {
 
 	public void validateWorkload(Duration monitorDuration) {
 		boolean unboundedMonitor = monitorDuration.toMillis() == Long.MAX_VALUE;
-		if (getEventsNum() == 0) {
-			// TPS mode
```

Comment:
Some parameters should also be removed? 
e.g. nexmark.metric.monitor.duration

Reply:
It also has been used in test, so I think we don't need to remove it now.